scripts/Connector/Connector_Hirose/conn_hirose_df13_tht_side.py

```diff
@@ -159,13 +159,11 @@ def generate_one_footprint(pins, configuration):
 
         kicad_mod.append(PolygoneLine(
             polygone=[
-                {'x': px,'y': py},
                 {'x': px-w,'y': py},
                 {'x': px-w,'y': py-l+0.25*w},
                 {'x': px,'y': py-l},
                 {'x': px+w,'y': py-l+0.25*w},
-                {'x': px+w,'y': py},
-                {'x': px,'y': py}],
+                {'x': px+w,'y': py}],
```

Comment:
Is this correct? Doesn't the polygon need to have the same start and end point? That's what I've seen. Did you try this?

Reply:
This generates a polyline with no errors. I've updated the generated footprints in the other PR in case you want to have a look at the result. Thanks!